pvlib/ivtools/sdm.py

```diff
@@ -203,24 +206,35 @@ def fit_desoto(v_mp, i_mp, v_oc, i_sc, alpha_sc, beta_voc, cells_in_series,
 
     References
     ----------
-    .. [1] W. De Soto et al., "Improvement and validation of a model for
+    .. [1] J. A Duffie, W. A Beckman, "Solar Engineering of Thermal Processes",
+       4th ed., Wiley, 2013. :doi:`10.1002/9781118671603`
+    .. [2] W. De Soto et al., "Improvement and validation of a model for
        photovoltaic array performance", Solar Energy, vol 80, pp. 78-88,
        2006. :doi:`10.1016/j.solener.2005.06.010`
+
     """
 
     # Constants
     k = constants.value('Boltzmann constant in eV/K')  # in eV/K
     Tref = temp_ref + 273.15  # [K]
 
     # initial guesses of variables for computing convergence:
-    # Values are taken from [2], p753
-    Rsh_0 = 100.0
-    a_0 = 1.5*k*Tref*cells_in_series
-    IL_0 = i_sc
-    Io_0 = i_sc * np.exp(-v_oc/a_0)
-    Rs_0 = (a_0*np.log1p((IL_0-i_mp)/Io_0) - v_mp)/i_mp
+    # Default values are taken from [2], p753
```

Comment:
Guessing this should be [1] now.  My copy of D&B (2nd edition) doesn't seem to have this procedure :(